### DISCUSSION TITLE
Unable to proceed without school email, but school doesn't provide them to students

### DISCUSSION THREAD

**Discussion by tylersb** [community/community#41801] [GitHub Education] [answered]
Hi,

When I tried applying for Github Education, my school (General Assembly) showed up in the database of schools that are covered by the program, however it blocked me from continuing the application unless I provided an email address with the school's domain. I spoke with my instructional team about it and they said that only GA staff, not students, receive an email address. Since the system thinks that I'm supposed to have an email address for this school as a student, I'm not even able to attempt to submit alternate forms of proof. When I reached out to support regarding this I got no response for multiple weeks, then the ticket was abuptly closed and I was directed here.

**Comment by samus-aran** [comment]
Hi @tylersb 👋 

First off I want to thank you for sharing your experiences and that we absolutely acknowledge your frustration. As we're encouraging our users to utilise the Community more and share knowledge on all things GitHub Global Campus sometimes, like this, users need explicit help. Let's see how we can assist. 

As you can imagine there are many types of academic verification and institutes around the world and our system is constantly evolving to accommodate this without setting the bar too high. There are also some institutes that supply their own passes and don't necessarily need students to go through the standard GitHub Student Application form - if they do they have strict requirements like having an academic email. I suspect your school is one of these. Please speak to your instructor and ask them how the institutes instructs their students to apply and whether you will receive an academic email or another form of validating your student status.

**Comment by tylersb** [comment, marked as answer]
Thank you :) I reached out to someone else on the staff who I hadn't spoken with yet and they let me know that they have a specific process for registering students with Github Education, however we just haven't reached that part of the course yet and we'll be setting it up in a week or two.

**Reply by samus-aran** [reply]
Fabulous to hear that @tylersb 🎉 Well done for reaching out to the faculty and finding a solution 👍

**Reply by CyberSecStudent2022** [reply]
Hello @tylersb, I was just curious if the issue with not being able to sign up for student benefits without a school assigned email address had been addressed/resolved yet?
I am very interested in this as I’m more of a hands on learner and would like to get some more experience in some of the topics we’re going over in class but the program for Cyber Security that I’m taking doesn’t have assigned email addresses.

**Reply by tylersb** [reply]
Hi @CyberSecStudent2022, yes, samus-aran mentioned in this thread that some schools supply the students with specific passes for access to github education rather than them being able to sign up individually and that was the case with my institution. Around halfway through my course everyone in my class received individual links to sign up with and we all did it at the same time.

**Reply by TalexDreamSoul** [reply]
If our teacher did not explain this problem, can there be other ways to solve it, such as learning letter network verification??

https://www.chsi.com.cn/

**Comment by Pulmonic** [comment]
Oh, I'm happy to find this thread. I got the same issue and it unfortunate me a lot at first. Everyone in my class got the permit on using Github Education without any problem, but I'm the one with it. And, honestly, I was so happy to hear that we finally start using GitHub for studying, because it was so boring before.  I just use sources to get ready for some of my study tasks and waste all my free time on useless things. I eventually started to use tools like [https://gradesfixer.com/free-essay-examples/hamlet/](https://gradesfixer.com/free-essay-examples/hamlet/) so I will have more time to focus on courses I like. Now I'm so motivated by this news.

**Comment by williamregal** [comment]
Exam dumps for the [Fortinet NSE4_FGT-7.2  exact questions answer](https://www.marks4sure.com/NSE4_FGT-7-2-exam.html) are collections of practice questions and solutions that closely resemble the format and content of the real test. They are extensively given by many websites and platforms on the internet. Candidates utilize our exam dumps as an additional study tool while they get ready for the NSE4_FGT-7.2 certification.

**Comment by uly24** [comment]
An essay is a structured piece of writing that presents an argument or explores a specific topic. Typically, it consists of an introduction, body https://www.phdresearchproposal.org/environmental-research-proposal-topics/ paragraphs, and a conclusion. The introduction provides background information and a thesis statement, while the body develops and supports the argument. Each paragraph focuses on a specific point, providing evidence and analysis. The conclusion summarizes the main points and reinforces the thesis. Essays can vary in style and purpose, ranging from persuasive and argumentative to informative and reflective. Effective essays showcase critical thinking, coherence, and well-reasoned perspectives within a specified word limit.

**Comment by samus-aran** [comment]
Hi everyone! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or the conversation is diverting away from the original topic. Thank you for helping us maintain a productive and tidy community for all our members.
